bitnami/harbor/templates/clair/clair-secret.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.clair.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "harbor.clair" . }}
+  labels:
+{{ include "harbor.labels" . | indent 4 }}
+type: Opaque
+data:
+  config.yaml: {{ tpl (.Files.Get "conf/clair.yaml") . | b64enc }}
```

Comment:
You are creating a secret with a configuration file that contains no passwords, could you explain the need to do this?.

Reply:
if you use an external database, yes, the password is in the source section:
`clair:
      database:
        type: pgsql
        options:
          source: "postgres://harbor:MYPASSWORD@my-postgresql:5432/clair?sslmode=disable"`

Reply:
There is also a `posgresql.postgresqlPassword` that should be use for that purpose.

Reply:
Please, take into account that I am using the externalDatabase feature, so I have to set `postgresql.enabled: false` to avoid installing stable/postgresql that I won't use. This `postgresql.postgresqlPassword` has no effect in my use case.